src/deno_dir.rs

```diff
@@ -109,21 +110,26 @@ impl DenoDir {
     self: &DenoDir,
     module_name: &str,
     filename: &str,
-  ) -> DenoResult<String> {
+  ) -> DenoResult<(String, msg::MediaType)> {
     let p = Path::new(filename);
+    let mut media_type_filename = filename.to_string();
+    media_type_filename.push_str(".mime");
+    let mt = Path::new(&media_type_filename);
```

Comment:
Can you add a comment to this section. Something along the lines of:

```
// We write a special ".mime" file into the .deno/deps directory along side the
// cached file, containing just the mime type.
```

Reply:
👍